README.md

```diff
@@ -181,32 +181,32 @@ Outside of AWS, you can [pass in your own SSH keys](https://github.com/bottleroc
 
 To enable the container, you can change the setting in user data when starting Bottlerocket, for example EC2 instance user data:
 
-```
+```toml
 [settings.host-containers.admin]
 enabled = true
 ```
 
 If Bottlerocket is already running, you can enable the admin container from the default [control container](#control-container) like this:
 
-```
+```sh
 enable-admin-container
 ```
 
 Or you can start an interactive session immediately like this:
 
-```
+```sh
 enter-admin-container
 ```
 
 If you're using a custom control container, or want to make the API calls directly, you can enable the admin container like this instead:
 
-```
+```sh
 apiclient set host-containers.admin.enabled=true
```

Comment:
I'm personally not a fan of adding syntax highlighting for these kind of short one offs (mostly because GitHub resolves individual _words_ with highlights, not whole commands). For example, here `set` gets highlighted since it is a keyword in bash:

![Screen Shot 2022-10-07 at 10 23 16 AM](https://user-images.githubusercontent.com/23109390/194601763-21305690-4455-41e9-8dea-bc75d4368d77.png)

But other commands with this format don't get this treatment since they don't contain keywords:

![Screen Shot 2022-10-07 at 10 27 06 AM](https://user-images.githubusercontent.com/23109390/194602233-fe2adce1-6f43-40ce-97f1-c7d64e73256b.png)

So, it's a bit awkward and creates it's own inconsistencies. Seems the two problem words are `set` and `exec`.

Reply:
I'll admit, I hesitated on that. In a lot of cases, the `sh` type really doesn't add a lot of value. I'll change those to `txt` since we know we don't want or need syntax highlighting.

Reply:
That seems fine to me as I'm used to seeing keywords highlighted in shell snippets.